src/ray/common/task/task_spec.h

```diff
@@ -121,15 +126,16 @@ class TaskSpecification : public MessageWrapper<rpc::TaskSpec> {
   /// Return the ObjectIDs of any dependencies passed by reference to this
   /// task. This is recomputed each time, so it can be used if the task spec is
   /// mutated.
-  ///
+  /// \param add_dummy_dependency whether add dummy object to dependencies or not.
```

Comment:
```suggestion
  /// \param add_dummy_dependency whether to add a dummy object in the returned objects.
```

Reply:
It seems that the comment is added on the wrong method. It should be added to `GetDependencies`.